OrbitQt/orbitmainwindow.cpp

```diff
@@ -242,6 +244,45 @@ void OrbitMainWindow::SetupCodeView() {
   OrbitCodeEditor::setFileMappingWidget(ui->FileMappingWidget);
 }
 
+//-----------------------------------------------------------------------------
+void OrbitMainWindow::ShowFeedbackDialog() {
+  QPointer<QDialog> feedback_dialog = QPointer{
+      new QDialog{this, Qt::WindowTitleHint | Qt::WindowCloseButtonHint}};
```

Comment:
This is basically a memory leak because this dialog is destroyed when the parent (OrbitMainWindow) is destroyed. That means every click on the feedback creates a new dialog in memory. I recommend to put the QDialog object on the stack and pass `nullptr` as its parent. That should fix it. All the other objects can remain on the heap.

Reply:
Ah, good point, thanks.

Reply:
Fixed.